Product/Production/Adapters/General/CONNECTAdminGUI/src/main/java/gov/hhs/fha/nhinc/admingui/managed/PropertyBean.java

```diff
@@ -94,14 +94,14 @@ public void onGatewayValueEdit(CellEditEvent event) {
                 new FacesMessage(FacesMessage.SEVERITY_INFO, "INFO", "Property value changed for " + selectedProp.getKey()
                     + " from " + oldValue + " to " + newValue + "."));
         } catch (PropertyAccessException ex) {
-            LOG.warn("Unable to update property: " + selectedProp.getKey());
+            LOG.warn("Unable to update property {}: {}", selectedProp.getKey(), ex.getLocalizedMessage(), ex);
```

Comment:
Leaving as warning level for now, to be consistent with other JSF exception log messages in our code.